app/assets/stylesheets/components/_figure.scss

```diff
@@ -1,11 +1,45 @@
 .app-c-figure {
+  @extend %contain-floats;
   @include responsive-bottom-margin;
 }
 
 .app-c-figure__image {
-  max-width: 100%;
+  display: block;
+  width: 100%;
+
+  @include media(tablet) {
+    float: left;
+    width: 50%;
+  }
+}
+
+.app-c-figure__figcaption-wrapper {
+  @include media(tablet) {
+    box-sizing: border-box;
+    float: left;
+    padding-left: $gutter-half;
+    width: 50%;
+  }
 }
 
 .app-c-figure__figcaption {
   @include core-16;
+
+  padding-top: $gutter-half;
+  width: 100%;
+
+  @include media(tablet) {
+    @include core-14;
+
+    border-top: 1px solid $border-colour;
+    display: block;
+    vertical-align: top;
+  }
+
+  svg {
```

Comment:
Could use a class for BEM

Reply:
Updated: https://github.com/alphagov/government-frontend/pull/541/commits/5b855b10987cabb340b0a3c37d37636f68e51faa